app/code/community/SomethingDigital/InfluxDb/Model/Measurement/Changelog.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * Sends data about status of Enterprise_Index changelogs
+ *
+ * measurement: changelog
+ * tag(s): name
+ * field(s): metadata_version, version
+ *
+ */
+class SomethingDigital_InfluxDb_Model_Measurement_Changelog
+    extends SomethingDigital_InfluxDb_Model_Measurement_Abstract
+    implements SomethingDigital_InfluxDb_Model_MeasurementInterface
+{
+    public function send()
+    {
+        $metadatas = Mage::getModel('enterprise_mview/metadata')->getCollection();
+        $data = array();
+        foreach ($metadatas as $metadata) {
+            $data[] = $this->line($metadata);
```

Comment:
I had the arguments reversed for `array_map` but yeah, I think it's prettier without being obtuse.